elements/rh-badge/test/rh-badge.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { expect, html } from '@open-wc/testing';
+import { createFixture } from '@patternfly/pfe-tools/test/create-fixture.js';
+import { RhBadge } from '@rhds/elements/rh-badge/rh-badge.js';
+
+const element = html`
+  <rh-badge></rh-badge>
+`;
```

Comment:
Leaving this comment for if we want to go ahead and add some tests for styling into the spec test file.  If not, please resolve.